adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -188,6 +197,7 @@ public void export(InvalidRowLengthNotice toExport) throws IOException {
                 .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.ERROR)
                 .setAltEntityRow(toExport.getRowIndex())
                 .setAltEntityValue(String.valueOf(toExport.getExpectedLength()))
+                .clearAltEntityId()
```

Comment:
Good catch! As we said together, replace this by a call to `clear()` at the beginning of the chain to clear the reused internal builder